Update reply-url.md (#96914)
Azure/azure-content: articles/active-directory/develop/reply-url.md

| Microsoft work or school accounts in any organization's Azure Active Directory (Azure AD) tenant | 256 | `signInAudience` field in the application manifest is set to either *AzureADMyOrg* or *AzureADMultipleOrgs* |
| Personal Microsoft accounts and work and school accounts | 100 | `signInAudience` field in the application manifest is set to *AzureADandPersonalMicrosoftAccount* |

> The maximan number of redirect URIS can not be raised for [security reasons](#restrictions-on-wildcards-in-redirect-uris). If your scenario requires more redirect URIs than the maximum limit allowed, consider the following [state parameter approach](#use-a-state-parameter) as the solution.

Reviewer: Rather than stating that it cannot be raised, we should just state that if you run into the max limits on redirect uris, the solution is to implement state parameter approach. 
So, it would be something like "The maximum limit for redirect uris on AAD app is [X] and MSA app is [Y]. If your scenario requires more redirect URIs than the maximum allowed limit, you should use the [state parameter approach](#use-a-state-parameter) as the solution."

Author: Sounds good. Let's just add 'If your scenario requires more redirect URIs than the maximum allowed limit, you should use the [state parameter approach](https://github.com/MicrosoftDocs/azure-docs/pull/96914#use-a-state-parameter) as the solution.' after the table. 

![image](https://user-images.githubusercontent.com/102836982/184007123-7ac23ca0-2967-4128-a2fb-a941c7209087.png)

Reviewer: Sounds good. thank you for updating it.

> The maximan number of redirect URIS can not be raised for [security reasons](#restrictions-on-wildcards-in-redirect-uris). If your scenario requires more redirect URIs than the maximum limit allowed, consider the following [state parameter approach](#use-a-state-parameter) as the solution.

Reviewer 2: ```suggestion
The maximum number of redirect URIS can't be raised for [security reasons](#restrictions-on-wildcards-in-redirect-uris). If your scenario requires more redirect URIs than the maximum limit allowed, consider the following [state parameter approach](#use-a-state-parameter) as the solution.
```
edits
